raiden/raiden_service.py

```diff
@@ -413,7 +413,9 @@ def install_and_query_payment_network_filters(
             )
 
             chain_id = self.chain.network_id
-            for event in self.blockchain_events.poll_blockchain_events():
+            for event in self.blockchain_events.poll_blockchain_events(
+                self.get_block_number(),
```

Comment:
this looks wrong, why are you using `get_block_number`?

also, please avoid multiline `for`s

Reply:
This call runs only on setup/init, and as we're passing the block number down to the StatelessFilter instance, here we pass the current known block number, to avoid letting the filters do a ton of calls to `blockNumber` just to get `latest` changes.